Makefile

```diff
@@ -0,0 +1,20 @@
+.PHONY: all glidebin glide deps install fmt test
+
+all: install
+
+glidebin:
+	go get -u github.com/Masterminds/glide
+
+glide: glidebin
+	glide install
+
+deps: glide
+
+install: deps
+	go install . ./cmd/...
+
+fmt:
+	go fmt ./... $$(go list ./... | grep -v '/vendor/')
+
+test:
```

Comment:
Do you have a preference for `make test` vs `make check`? FWIW, this is only running the unit tests, the integration tests can be run using `go test -test.v -tags rpctest -logoutput` on the condition that the latest binary is already installed.

Reply:
Another alternative is that `make test` just runs the unit tests, and `make check` could run the itests after the unit tests succeed.